keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -333,3 +340,36 @@ def _ensure_inputs_are_compute_dtype(self, inputs):
                 self.compute_dtype,
             )
         return inputs
+
+    def _pad_bounding_boxes(self, bounding_boxes):
```

Comment:
lets make this  a stable API in `keras_cv/bounding_box` under utilities.

Reply:
sounds good!

Reply:
look a different approach here like below
```
rt = tf.ragged.constant([[[1,0],[2,0],[3,0]],[[2,0],[0,3]],[[1,0]]])
# convert to dense tensor
dt = rt.to_tensor()
# drop padded
tf.RaggedTensor.from_tensor(dt, lengths=rt.nested_row_lengths())
```

Reply:
done